spyder/widgets/classfunctiondropdown.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
```

Comment:
Move this file to panels folder https://github.com/spyder-ide/spyder/tree/master/spyder/widgets/panels